arbos/parse_l2.go

```diff
@@ -162,12 +163,10 @@ func parseL2Message(rd io.Reader, poster common.Address, timestamp uint64, reque
 			}
 			nestedSegments, err := parseL2Message(bytes.NewReader(nextMsg), poster, timestamp, nextRequestId, chainId, depth+1)
 			if err != nil {
-				log.Warn("Failed to parse L2Message in a batch")
-			}
-			if nestedSegments != nil && nestedSegments.Len() > 0 {
-				segments = append(segments, nestedSegments...)
```

Comment:
Does removing this break backwards compatibility?

Reply:
This change reverts this piece of code to what it was in arbitrum